tmp/2025-05-28-tychi-logs.txt

```diff
@@ -0,0 +1,156 @@
+    ~/SourceCode/wasup    main ?2  wasup ./README.md $WAS/space/$SPACE/README.md --identity ~/.ssh/id_ed25519_was_test
```

Comment:
did you export your vars?

```shell
/tmp 
⚡ VAR=1    

/tmp 
⚡ ./script
VAR=

/tmp 
⚡ export VAR=1

/tmp 
⚡ ./script    
VAR=1
```

Reply:
you can also do like `wasup ./README.md https://storage.bengo.is/space/:uuid`
(you dont have to use the vars)